snowplow_tracker/emitters.py

```diff
@@ -53,7 +53,7 @@ def __init__(
             endpoint: str,
             protocol: HttpProtocol = "https",
             port: Optional[int] = None,
-            method: Method = "get",
+            method: Method = "post",
```

Comment:
Can't comment in there directly but please also make sure to change the default method to POST in the `CeleryEmitter`, see `celery/celery_emitter.py`.